### ISSUE TITLE
"An existing connection was forcibly closed by the remote host"

### ISSUE THREAD

**Issue by Kunderao** [finnhubio/Finnhub-API#259]
## What is the issue ? Plz describe in details

Finnhubs response to "An existing connection was forcibly closed by the remote host"

"The server needs to close websocket so we can restart and apply new updates from time to time. Plz handle disconnection and reconnect accordingly."

How exactly should this disconnection be handled in Go. Should we just reissue web socket Dial and start over?

**Comment by finnhubio** [comment]
Yes you should simply drop the previous connection and open a new one. Putting the websocket inside an infinite for loop is 1 way to do it.
